services/movie_session.py

```diff
@@ -26,11 +27,11 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
 
 def update_movie_session(
     session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+    show_time: Optional[str] = None,
+    movie_id: Optional[int] = None,
+    cinema_hall_id: Optional[int] = None,
 ) -> None:
-    movie_session = MovieSession.objects.get(id=session_id)
+    movie_session: MovieSession = MovieSession.objects.get(id=session_id)
```

Comment:
Consider handling exceptions for the `get` method in `update_movie_session` to manage cases where a movie session with the given ID does not exist. You can use `MovieSession.DoesNotExist` to catch such exceptions and handle them appropriately.